.github/workflows/welcome-new-contributors.yaml

```diff
@@ -0,0 +1,28 @@
+name: Welcome new contributors
+
+on:
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  welcome:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/first-interaction@v3
```

Comment:
Do you know how it detects the first interaction ? Does it check how many PRs user opens ?